assets/css/app.css

```diff
@@ -137,6 +137,13 @@
     background-size: 50%;
     background-position: left -30% top 200px, right -30% bottom -15%;
   }
+
+  .room-page {
```

Comment:
Can we name it so it is known this class changes the background? Something like `.room-page-bg` I feel like this would be more idiomatic with the tailwind style